src/Wizard/Wizard.component.js

```diff
@@ -98,7 +124,15 @@ const Wizard = React.createClass({
             .filter(({step}) => step.visible !== false);
     },
 
-    _renderButtons(step, showPrevious, showNext) {
+    _renderButtons(step, showPrevious, showNext, showHelp) {
+        const renderHelp = () => (
+            <ToolbarGroup>
```

Comment:
Indentation is missing here...

Reply:
Sorry, I don't see where it's missing...

```
1. _renderButtons(step, showPrevious, showNext, showHelp) {
2.    const renderHelp = () => (
3.        <ToolbarGroup>
```

Line 2 starts with 4 extra spaces, and line 3 (the next level) has 4 more.

Reply:
Cool, I see it now, thanks!